src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>
```

Comment:
In case you remove the React import as suggested above, you can import StrictMode instead.
```suggestion
import {StricMode} from "react"
  <StrictMode>
    <App />
  </StrictMode>
```